source/develop/release-management/features/infra/aaa_faq.html.md

```diff
@@ -0,0 +1,107 @@
+you add following lines into /etc/ovirt-engine/aaa/your_profile.properties:
+
+```
+sequence-init.init.100-my-basedn-init-vars = my-basedn-init
+sequence.my-basedn-init.010.description = set custom baseDN
+sequence.my-basedn-init.010.type = var-set
+sequence.my-basedn-init.010.var-set.variable = simple_baseDN
+sequence.my-basedn-init.010.var-set.value = dc=your,dc=different,dc=dn
+```
+
+### What's default connection/response timeout for LDAP? Can I change it?
+
+Default connection timeout is 5 seconds.
+Default response timeout is 60 seconds.
+
+In order to change these, you need to change following configuration options:
+
+```
+# Values are in miliseconds
+pool.default.connection-options.connectTimeoutMillis = 300
+pool.default.connection-options.responseTimeoutMillis = 30000
+```
+
+### Is it possible to use forest trust in Active Directory setup?
+
+It is not currently supported.
+
+### Is it possible to use multi-domain Active Directory setup?
+
+Yes, it's fully supported since oVirt 3.5. All domains are automatically resolved. User only need to specify Active Directory forest name.
+
+### Is it possible to change default domain of multi-domain Active Directory setup?
+
+Yes, you can configure it with aaa-misc extension. First you need to install it:
+
+```
+yum install -y ovirt-engine-extension-aaa-misc
+```
+
+Then create such *mapping* configuration, for example */etc/ovirt-engine/extensions.d/mapping.properties*
+
+```
+ovirt.engine.extension.name = mapping
+ovirt.engine.extension.bindings.method = jbossmodule
+ovirt.engine.extension.binding.jbossmodule.module = org.ovirt.engine-extensions.aaa.misc
+ovirt.engine.extension.binding.jbossmodule.class = org.ovirt.engineextensions.aaa.misc.mapping.MappingExtension
+ovirt.engine.extension.provides = org.ovirt.engine.api.extensions.aaa.Mapping
+config.mapUser.type = regex
+config.mapUser.regex.pattern = ^(?<user>[^@]*)$
+config.mapUser.regex.replacement = ${user}@<default.suffix>
+config.mapUser.regex.mustMatch = false
+```
+
+Replace *<default.suffix>* with the desired suffix, for example *${user}@subdomain.domain.com*.
+Then apply above mapping into *authn* extension:
+
+```
+ovirt.engine.aaa.authn.mapping.plugin = mapping
+```
+
+Now, when you insert into login field user with username *joe* it will be mapped to *joe@subdomain.domain.com*.
+
+### Is it possible to use client certificate authentication?
+
+Yes, it's possible with *apache mod_ssl* module. Full blog post is available [here](http://machacekondra.blogspot.cz/2016/02/client-certificate-authentication-with.html).
+
+### Is it possible to use CAS for authentication?
+
+Yes, it's possible with *apache mod_auth_cas* module. Full blog post is available [here](http://www.ovirt.org/blog/2016/04/sso/).
+
+### Is it possible to use SSSD for authentication?
+
+Unfortunatelly it's not supported, yet. You can track [this](https://bugzilla.redhat.com/show_bug.cgi?id=829292) bugzilla for more information.
+
+<Category:Feature>
```

Comment:
Looks good to me. @sandrobonazzola, @thatdocslady, @mykaul  could you please merge?